tests/e2e.test.ts

```diff
@@ -384,5 +385,45 @@ describe('End-to-end tests', () => {
         `)
       })
     })
+    describe('--error-format option', () => {
+      it('fails with an invalid error format', async () => {
+        const invalidErrorFormat: string = 'invalid-format'
+
+        const checkOutput = await cli(
+          `check --error-format ${invalidErrorFormat}`,
+          basicTsErrorOutput
```

Comment:
The output is not really needed for the test case, since the command should fail before it is even evaluated.

You could remove it to avoid any influence on the test outcome.

Reply:
I removed basicTsErrorOutput.